exchanges/alphapoint/alphapoint_wrapper.go

```diff
@@ -98,24 +105,28 @@ func (a *Alphapoint) GetExchangeHistory(p pair.CurrencyPair, assetType string) (
 
 // SubmitExchangeOrder submits a new order and returns a true value when
 // successfully submitted
-func (a *Alphapoint) SubmitExchangeOrder(p pair.CurrencyPair, side string, orderType int, amount, price float64) (int64, error) {
-	return a.CreateOrder(p.Pair().String(), side, orderType, amount, price)
+func (a *Alphapoint) SubmitExchangeOrder(base, quote pair.CurrencyItem, side exchange.OrderSide, orderType exchange.OrderType, amount, price float64, clientID string) (int64, error) {
```

Comment:
As discussed, it would be better to use a pair.CurrencyPair variable just like we do with the ticker and orderbook, that way we can use the exchange.FormatExchangeCurrency to line up with the exchanges expected request and would save time not having to adjust it to match the exchange.